main.py

```diff
@@ -246,26 +249,49 @@ def match_flows(self, switch, args, stored_flows, many=True):
             return None
         return response
 
-    # pylint: disable=redefined-outer-name
-    # pylint: disable=too-many-branches
+    def process_tables(self, switch, table_id, args, stored_flows, actions):
+        """Resolve the table context and get actions in the matched flow"""
+        goto_table = False
+        actions_ = []
+        flow = self.match_flows(switch, table_id, args, stored_flows, False)
+        if flow and 'actions' in flow['flow']:
+            actions_ = flow['flow']['actions']
+        elif flow and 'instructions' in flow['flow']:
+            for instruction in flow['flow']['instructions']:
+                if instruction['instruction_type'] == 'apply_actions':
+                    actions_ = instruction['actions']
+                elif instruction['instruction_type'] == 'goto_table':
+                    table_id_ = instruction['table_id']
+                    if table_id < table_id_:
+                        table_id = table_id_
+                        goto_table = True
+                    else:
+                        msg = f"Wrong table_id: \
+                            A packet can only been directed to a \
+                                flow table number greather than {table_id}"
+                        log.error(msg)
+                        raise ValueError(msg) from ValueError
+        actions.extend(actions_)
+        return flow, actions, goto_table, table_id
+
     def match_and_apply(self, switch, args, stored_flows):
-        # pylint: disable=bad-staticmethod-argument
         """Match flows and apply actions.
         Match given packet (in args) against
         the stored flows (from flow_manager) and,
         if a match flow is found, apply its actions."""
-        flow = self.match_flows(switch, args, stored_flows, False)
+        table_id = 0
+        goto_table = True
         port = None
         actions = []
-        # pylint: disable=too-many-nested-blocks
+        while goto_table:
+            try:
+                flow, actions, goto_table, table_id = self.process_tables(
+                    switch, table_id, args, stored_flows, actions)
+            except ValueError as exception:
+                raise exception
```

Comment:
This try except here isn't doing much.

You'll need to handle it as close as possible to the request context for instance when `self.tracepath` is called here https://github.com/kytos-ng/sdntrace_cp/blob/master/main.py#L67 and https://github.com/kytos-ng/sdntrace_cp/blob/master/main.py#L82, and then from that point reraise a `HTTPException(409, str(exc))`

Reply:
Ok.